app-scroll-effects/app-scroll-effects-behavior.html

```diff
@@ -14,22 +14,25 @@
 
 <script>
   /**
-   * `Polymer.AppScrollEffectsBehavior` provides an interface that allows an element to use scrolls effects.
+   * `Polymer.AppScrollEffectsBehavior` provides an interface that allows an
+   * element to use scrolls effects.
    *
    * ### Importing the app-layout effects
    *
-   * app-layout provides a set of scroll effects that can be used by explicitly importing
-   * `app-scroll-effects.html`:
+   * app-layout provides a set of scroll effects that can be used by explicitly
+   * importing `app-scroll-effects.html`:
    *
    * ```html
-   * <link rel="import" href="/bower_components/app-layout/app-scroll-effects/app-scroll-effects.html">
+   * <link rel="import"
+   * href="/bower_components/app-layout/app-scroll-effects/app-scroll-effects.html">
```

Comment:
This _might_ be an issue with the README generator (i.e. will it remove the `* ` in the HTML block), but maybe it's ok?

Reply:
I can tell it to not reflow comments

Reply:
Sheesh, nevermind, that's a bad bad flag
![image](https://user-images.githubusercontent.com/5981958/38588331-1720bcd8-3cda-11e8-940c-aef5fda99568.png)

Reply:
I think it's better if it enforces the line limits on all comments - code snippets in comments are a small use case, and I'm not even sure if the README generator is used on this repo anyways. Suggest you ignore this issue for now, and we'll deal with it if it's an issue later.